api/turing/api/ensembling_job_api.go

```diff
@@ -50,7 +50,7 @@ func (c EnsemblingJobController) Create(
 		return BadRequest("only pyfunc ensemblers allowed", fmt.Sprintf("ensembler type given: %T", v))
 	}
 
-	ensemblingJob, err := c.EnsemblingJobService.CreateEnsemblingJob(job, projectID, pyFuncEnsembler)
+	ensemblingJob, err := c.EnsemblingJobService.CreateEnsemblingJob(job, project, pyFuncEnsembler)
```

Comment:
Hmm this makes things quite awkward. From what I understand from the docs, the implementation would look something like this:

```go
func (j *EnsemblingJob) AfterFind() error {
  proj, err := MLPServiceSingleton.GetProject(j.ProjectID) // <------- this is where it gets awkward, because we need the project name
  if err != nil {
    return err
  }
  if err := s.populateMonitoringURL(job, project); err != nil {
    return err
  }
  return nil
}
```

Since it doesn't make sense to instantiate a service inside a model, a MLPService singleton has to be used.

The other way is to store the project name inside ensembling job which probably makes the most sense. Do you know if there are assumptions that MLP project name will always remain the same? Or can a project be renamed?

Reply:
Currently project can't be renamed, but I wouldn't rely on this assumption going forward. Yeah, having a singleton service would be weird. Okay, ignore this comment then

Reply:
Ok but I will revert this to take in projectID instead of project instead.